wdfn-server/waterdata/services/ogc.py

```diff
@@ -0,0 +1,35 @@
+"""
+Helpers to retrieve OGC data - Networks.
+"""
+from waterdata import app
+from waterdata.utils import execute_get_request
+
+
+def get_networks(network_cd):
+    """
+    Gets the network data from a json file
+    :param network_cd: collections-id
+    """
+    # Handle feature flag for network pages
+    if not app.config['NETWORK_PAGES_ENABLED']:
+        return []
+
+    url = app.config['OGC_SERVICE'].format('json')
+
+    if network_cd != 'ALL':
+        url = url +'/'+network_cd+'/'
+    url = url + '?f=json'
+
+    #print('services: ogc.py: get_networks(): ' + url)
+    response = execute_get_request(url)
+
+    if response.status_code != 200:
+        return []
+    try:
+        resp_json = response.json()
+    except ValueError:
+        return []
+    else:
+        if network_cd == 'ALL':
```

Comment:
I think the whole object should be returned rather than just the collection part. This service method may be used for different purposes in the future and returning the whole object make the method more reusable. Then we can eliminate this else clause.

Reply:
I was noticing that the return is different if you are requesting a specific network as opposed to all networks.  The collection part in the json is not there if you only ask for a specific network.  I wonder if the call itself should be changed to look the same regardless of whether you're asking for one network or all?

Reply:
That's what I'm thinking. Just return the entire response which will always be a dictionary.